packages/core/src/components/Table/TableVirtualizedBody/TableVirtualizedBody.tsx

```diff
@@ -95,12 +120,22 @@ const TableVirtualizedBody = forwardRef(
               <List
                 itemSize={RowHeights[size]}
                 height={height}
-                itemCount={items.length}
+                itemCount={virtualizedWithHeader ? items.length + 1 : items.length}
                 width={width}
                 onScroll={handleVirtualizedVerticalScroll}
                 outerRef={element => {
                   virtualizedListRef.current = element;
                 }}
+                innerElementType={
+                  virtualizedWithHeader
+                    ? forwardRef(({ children, ...rest }, ref) => (
+                        <div ref={ref} {...rest}>
+                          {headerRenderer(columns)}
+                          {children}
+                        </div>
+                      ))
+                    : undefined
+                }
```

Comment:
is this just lets you wrap everything that VirtualizedList is about to render?
also, do we absolutely need the wrapping div? and what's `rest`? is it coming from VirtualizedList?

Reply:
Can we try Fragments there maybe?
or is it a must? (a yes is a legitimate answer 🤓)

Reply:
The div exists in the default behavior of VirtualizedList, so we need to include it here to maintain the current structure. The rest props come from VirtualizedList